api/v3/Activity.php

```diff
@@ -482,6 +476,14 @@ function _civicrm_api3_activity_get_formatResult($params, $activities) {
         }
         break;
 
+      case 'case_id':
```

Comment:
Believe me I tried. The query function just doesn't want to join across 2 tables like that.
My frustration with this bit was what prompted my comments about joins over in the api4 discussion.